modules/tinymce/src/themes/silver/main/ts/ui/dialog/TextField.ts

```diff
@@ -78,10 +78,12 @@ const renderTextField = (spec: TextField, providersBackstage: UiFactoryBackstage
 
   const placeholder = spec.placeholder.fold( Fun.constant({}), (p) => ({ placeholder: providersBackstage.translate(p) }));
   const inputMode = spec.inputMode.fold(Fun.constant({}), (mode) => ({ inputmode: mode }));
+  const name = { 'data-mce-name': spec.name };
 
   const inputAttributes = {
     ...placeholder,
-    ...inputMode
+    ...inputMode,
+    ...name
```

Comment:
You don't need spread for an object property that will always exist 🤨 
```suggestion
    'data-mce-name': spec.name
```